### PR TITLE
Solves Issue No. 728- Unneccesary redirecting to a seperate 'Rate us' and 'Contact Us' + Routing of the navbar.

### PR DESCRIPTION
**Pull request by Nidhi-Satyapriya**
## Description
I made the following corrections:-
1) On clicking "Rate Us" and "Contact Us", the control directs to their respective sections, ensuring consistency of the website.
2) Introduced a confirmatory message in "#contact" section (disappears after 5secs) + beautified css in the same.
3) Added a page named "med.html". It serves as the order portal of the "service" tab.
4) Proper Routing from "news.html" ,"service.html" and "Doctor Experience.html" to rest of the pages and sections.
5) Proper routing from the footer of all the pages.
6) Introduced smooth scrolling to the different sections of the page, feature enhancement.
7) Removed the "rateus.html" and "contact.html" , since its utility was already satisfied.

## Related Issues
Solves Issue No. 728- Unneccesary redirecting to a seperate 'Rate us' and 'Contact Us' + Routing of the navbar.
- Closes https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/issues/728

## Type of PR

- [X] Bug fix
- [X] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)

https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/assets/123321320/072c852d-14fa-43eb-bf26-a2f9b273da87
[Maps works fine. Network problem here].
1) Scrolls down to the respective sections.
2) On dropping a msg, displays a confirmatory message too.

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
 I noticed that there are still some pages left to be created to be linked to the footer. Eg- Testimonials. Users looking for creating issues can work on this.

### CODE REVIEW

**Comment by varshith257** [comment]
@Nidhi-Satyapriya Contact us link in the navbar has been broken:

image

**Comment by Nidhi-Satyapriya** [comment]
@varshith257 yes, happened while solving merge conflicts. I resolved it. Would you check once?

**Review by varshith257 — APPROVED** [review]

File: med.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (varshith257) [inline med.html:1]: Is this file related to this PR? I don't think so

Comment (Nidhi-Satyapriya) [reply med.html:1]: Sir,  in the 'services' tab, I observed that for placing orders, the previous developers had mentioned a section called "med.html", which didn't lead anywhere. So to fix this, I created this page, facilitating proper routing.

**Comment by Nidhi-Satyapriya** [comment]
Sir, is my job done and the issue is closed or I need to look over it once again? @varshith257

File: med.html (hunk 1 above)
Comment (varshith257) [reply med.html:1]: @Nidhi-Satyapriya Good to keep it in then

**Comment by varshith257** [comment]
@Nidhi-Satyapriya Please resolve conflicts and LGTM

**Comment by Nidhi-Satyapriya** [comment]
@varshith257 I just resolved the conflicts. You may know review this.

File: Html-Files/news.html [hunk 2]
```diff
@@ -632,32 +632,33 @@ <h3>RapiDoc's Research Team Publishes Groundbreaking Study on Preventive Healthc
...
                   <h4>Legal</h4><br>
                   <ul class="links">
                     <li><a href="#">Customer Agreement</a></li>
-                    <li><a href="#">Privacy Policy</a></li>
+                    <li><a href="/Html-Files/privacy_policy_page.html">Privacy Policy</a></li>
+                    <li><a href="/Html-Files/terms_and_conditions.html">Terms and Conditions</a></li>
                     <li><a href="#">GDPR</a></li>
                     <li><a href="#">Security</a></li>
-                    <li><a href="#">Testimonials</a></li>
+                    <li><a href="/index.html#test">Testimonials</a></li>
```
Comment (varshith257) [inline Html-Files/news.html]: Can be #test? It's now redirecting to top of home page

Comment (Nidhi-Satyapriya) [reply Html-Files/news.html]: @varshith257  test.html is for blood tests, not testimonials. People were overlooking it and directing to the wrong page. I have made this correction almost everywhere.
A dedicated page for testimonials is yet to be made.
